### PR TITLE
Add filters

### PR DESCRIPTION
**Pull request by nerimartinez**

### CODE REVIEW

File: components/src/widgets/complexTable/widget.vue [hunk 1]
```diff
@@ -1,4 +1,46 @@
 <template>
+  <ui-menu>
+    <ui-button
+      slot="trigger"
+      :disabled="filterableHeaders.length === 0"
```
Comment (arnaugiralt) [inline components/src/widgets/complexTable/widget.vue:5]: Could be a computed property

File: components/src/widgets/complexTable/widget.vue [hunk 2]
```diff
@@ -72,13 +120,25 @@ const props = defineProps({
...
 
+const filtersApplied = reactive({})
+
 const totalPages = computed(() => Math.ceil(props.totalItems / ITEMS_PER_PAGE))
 const previousButtonDisabled = computed(() => props.currentPage === 1)
 const nextButtonDisabled = computed(() => props.currentPage === totalPages.value)
+const filterableHeaders = computed(() => props.headers.filter(header => header.filterable))
 
-const previousClicked = () => emit('previousClicked');
+const cleanFiltersApplied = computed(() => Object.keys(filtersApplied).reduce(function(obj, k) {
```
Comment (arnaugiralt) [inline components/src/widgets/complexTable/widget.vue:130]: Why isn't the reduce callback an arrow function?

**Review by arnaugiralt — APPROVED** [review]

File: components/src/widgets/complexTable/widget.vue (hunk 1 above)
Comment (nerimartinez) [reply components/src/widgets/complexTable/widget.vue:5]: yeap, but it's just one condition

File: components/src/widgets/complexTable/widget.vue (hunk 2 above)
Comment (nerimartinez) [reply components/src/widgets/complexTable/widget.vue:130]: just because, there is no any specific reason, why do you want an arrow function?

Comment (arnaugiralt) [reply components/src/widgets/complexTable/widget.vue:130]: I find arrow functions more concise, nothing else. The question for me is the reverse, why use a regular function?

**Review by arnaugiralt — APPROVED** [review]

**Review by theConscience — APPROVED** [review]

**Pull request merged**